.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: clojure
+lein: 2.9.1
 install:
-- "pushd kibit"
-- "lein deps"
-- "popd"
+- "LEIN_USE_BOOTCLASSPATH=no lein cache-version"
```

Comment:
I think `LEIN_USE_BOOTCLASSPATH=no` could be set once as an environment variable?

Reply:
Also, if we are going to add a makefile, can we use those make commands here?